docs/DataDictionary.md

```diff
@@ -105,19 +105,19 @@ These are used for configuring a testing scenario.  `LoadTestConfig` will contai
 | EntityType      |   String   | Entity type used for filtering  | Yes | [`ClientStatus`, `LoadTestConfig`, `TestRun`] |
 | Id              |   String   | GUID used to retrieve the object directly. | Yes | |
 | Name            |   String   | Friendly name so that users may more easily identify configs | No | |
-| Files           |  String[]  | List of files to test   |   Yes     | match `--files` CLI flag |
+| Files           |  String[]  | List of files to test (default: `Null`)  |   Yes     | match `--files` CLI flag |
 | StrictJson      |   Boolean  | Use strict json when parsing (default: `False`) | No | match to `--strict-json` CLI flag |
-| BaseURL         |   String   | Base url for files (default is empty) | No | match to `--base-url` CLI flag |
-| VerboseErrors   |   Boolean  | Displays validation error messages | No | match to `--verbose-errors` CLI flag |
-| Randomize       |   Boolean  | Requires `RunLoop=True`. Dictates whether to process a load file top to bottom (default: `false`) or randomly | No | match to `--random` CLI flag |
-| Timeout         |    Int     | Request timeout in seconds (default: 30) | No | match to `--timeout` CLI flag |
-| Server          |  String[]  | Server(s) to test (default is empty) | Yes | match to `--server` CLI flag |
-| Tag             |   String   | Tag for log | No | match to `--tag` CLI flag |
-| Sleep           |   String   | Sleep (ms) between each request (default: 0) | No | match to `--sleep` CLI flag |
-| RunLoop         |   Boolean  | Run test in an infinite loop (default: False) | No | match to `--run-loop` CLI flag |
-| Duration        |    Int     | Requires `RunLoop=True`. Test duration (seconds) (default: 0) | No | match to `--duration` CLI flag |
-| MaxErrors       |    Int     | Requires `RunLoop=False`. Max validation errors (default: 10) | No | match to `--max-errors` CLI flag |
-| DryRun          |   Boolean  | Validate the settings with the target clients (default `false`) | No | match to `--dry-run` CLI flag |
+| BaseURL         |   String   | Base url for files (default: `Null`) | No | match to `--base-url` CLI flag |
+| VerboseErrors   |   Boolean  | Displays validation error messages (default: `False`) | No | match to `--verbose-errors` CLI flag |
+| Randomize       |   Boolean  | Requires `RunLoop=True`. Dictates whether to process a load file top to bottom (default: `False`) or randomly | No | match to `--random` CLI flag |
+| Timeout         |    Int     | Request timeout in seconds (default: `30`) | No | match to `--timeout` CLI flag |
+| Server          |  String[]  | Server(s) to test (default: `Null`) | Yes | match to `--server` CLI flag |
```

Comment:
I think this is for LodeRunner in client mode, and the default shouldn't be null because a value is required

Reply:
I removed (default: `Null`) for Server description